manual/cops_sorbet.md

```diff
@@ -501,7 +501,7 @@ end
 
 Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
 --- | --- | --- | --- | ---
-Enabled | Yes | Yes  | - | -
```

Comment:
This was missed in #131, but should have been a giveaway that something was wrong.